easyrsa3/easyrsa

```diff
@@ -941,34 +949,47 @@ Matching file found at: "
 		die "Failed to sign '$name'"
 	}
 
-	# inline it
-	if [ $EASYRSA_INLINE ]; then
-		inline_creds
-	fi
 } # => build_full()
 
-# Create inline credentials file for this node
-inline_creds ()
+# Create inline credentials file
+build_inline ()
 {
-	[ -f "$EASYRSA_PKI/$EASYRSA_REQ_CN.creds" ] \
-	&& die "Inline file exists: $EASYRSA_PKI/$EASYRSA_REQ_CN.creds"
+	# TODO: Improve build_inline integration. eg: renew/revoke and ta.key
+	verify_ca_init
+	name="$1"
+	ca_file="$EASYRSA_PKI/ca.crt"
+	crt_file="$EASYRSA_PKI/issued/$name.crt"
+	key_file="$EASYRSA_PKI/private/$name.key"
+	out_file="$EASYRSA_PKI/inline/$name.conf"
+
+	[ -f "$out_file" ] && [ ! "$EASYRSA_BATCH" ] \
```

Comment:
Why aren't you using a traditional conditional with if/then here?  This should be standard throughout.

Reply:
Not a problem.